### DISCUSSION TITLE
How to zip my password protected file so it can be downloaded on any computer

### DISCUSSION THREAD

**Discussion by gaijinfellow** [aonez/Keka#1478] [Q&A] [answered]
I want to zip a password protected folder with Keka for the purpose of being a paid  digital download that can be opened by anyone with the password on Windows, Mac, or Chromebook. What is the best way to Zip a password proteced file so that anyone can download it, just zip it as a standard Zip with password protection and that will work?

**Comment by aonez** [comment, marked as answer]
Indeed, ZIP is the most common and supported format. Just be sure to **not** use AES256 since it is not as widely supported as the legacy encryption method,

**Reply by gaijinfellow** [reply]
Okay, great! Thanks.
